templates/zerver/realm_creation_link_invalid.html

```diff
@@ -0,0 +1,25 @@
+{% extends "zerver/portico.html" %}
+
+{% block title %}
+<title>{{ _("Organization creation link expired or invalid") }} | Zulip</title>
+{% endblock %}
+
+{% block portico_content %}
+<div class="error_page">
+    <div class="container">
+        <div class="row-fluid">
+            <img class="hourglass-img" src="/static/images/errors/timeout_hourglass.png" alt=""/>
+            <div class="errorbox">
+                <div class="errorcontent">
+                    <h1 class="lead">{{ _("Organization creation link expired or invalid") }}</h1>
+                    <p>
+                        {% trans %}
+                        Unfortunately, this is not a valid link for creating an organization. Please <a href="/new/">obtain a new link</a> and try again.
```

Comment:
I feel like /new/ is not the right URL if OPEN_REALM_CREATION is False, which might be the more common way to get to this page.

One idea would be to have an if/else in the caller that decides what link URL to pass to this, with the other being the little section on ReadTheDocs for how to get a new creation link.